README.md

```diff
@@ -101,117 +64,45 @@ We may leave the zookeeper server now:
 quit
 ```
 
-Create the `backbeat-replication` kafka topic:
-
-```
-kafka-topics --create \
---zookeeper localhost:2181/backbeat \
---replication-factor 1 \
---partitions 1 \
---topic backbeat-replication
-```
-
 ### Run Scality Components
 
-#### Start Vault and Scality S3 servers
-
-Start the Vault server (this requires access to the private Vault repository):
-
-```
-git clone https://github.com/scality/Vault ~/replication/vault && \
-cd ~/replication/vault && \
-yarn i && \
-chmod 400 ./tests/utils/keyfile && \
-VAULT_DB_BACKEND=MEMORY node vaultd.js
-```
-
-In a new shell, start the Scality S3 server:
+#### Start Vault and Scality Cloudserver
```

Comment:
By using the `"type":"account"` method of authentication, vault should not be needed. Moreover since Vault is a private component like stated, to me it does not make much sense in an open source guide, unless the user is already a Scality customer. I believe we could instead document a true open source setup without Vault, with backbeat using account credentials to access the source and target.